File: source/BrandonWilliamsCs.EuchreNight/BrandonWilliamsCs.EuchreNight.WebApi/HandReports/PlaceHandReportDto.cs
namespace BrandonWilliamsCs.EuchreNight.WebApi.HandReports;

public class PlaceHandReportDto
{
  public required string SessionId { get; set; }
  public required int RoundNumber { get; set; }
  public required int TableNumber { get; set; }
  public required int HandNumber { get; set; }
  public required int CallingPlayerNumber { get; set; }
  public required bool CallerWentAlone { get; set; }
  public required IList<int> WinningPlayerNumbers { get; set; }
  public required bool WinnersTookAllTricks { get; set; }
}

File: source/BrandonWilliamsCS.EuchreNight.WebApp/src/api/PlaceHandReportDto.ts
export interface PlaceHandReportDto {
  sessionId: string;
  roundNumber: number;
  tableNumber: number;
  handNumber: number;
  callingPlayerNumber: number;
  callerWentAlone: boolean;
  winningPlayerNumbers: number[];
  winnersTookAllTricks: boolean;
}
